modules/470-chrony/images/chrony/werf.inc.yaml

```diff
@@ -1,58 +1,72 @@
 {{- $version := "4.4" }}
 ---
+artifact: {{ $.ModuleName }}/{{ $.ImageName }}-src-artifact
+fromArtifact: src-artifact
+git:
+- add: /{{ $.ModulePath }}modules/470-{{ $.ModuleName }}/images/{{ $.ImageName }}/
+  to: /src/config-builder
+  includePaths:
+  - '**/*.go'
+  - '**/*.mod'
+  - '**/*.sum'
+  stageDependencies:
+    install:
+    - '**/*.go'
+    - 'go.mod'
+    - 'go.sum'
+shell:
+  beforeInstall:
+  - git clone --depth 1 --branch {{ $version }} {{ $.SOURCE_REPO }}/chrony/chrony.git /src/chrony
+---
 artifact: {{ .ModuleName }}/build-chrony-static-artifact
 from: {{ .Images.BASE_ALPINE_DEV }}
```

Comment:
Please do not use DEV image, uese BASE_ALPINE with proxy

Reply:
Which proxy are you talking about? chrony build has all of its code bundled in git, go build is already using GOPROXY

Reply:
alpine packages proxy. If you cannot install any packages proxy isn't needed.

Reply:
No packages are installed, base image changed, so, it is done